app/lint.xml

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <lint>
     <!-- Disable the "Using Log instead of Timber" lint warnings -->
-    <issue id="LogNotTimber" severity="ignore" />
+    <issue id="LogNotTimber" severity="ignore">
+        <!-- Add specific paths to ignore -->
+        <ignore path="src/org/commcare/utils/FileUtil.java" />
+        <ignore path="src/org/commcare/activities/components/ImageCaptureProcessing.java" />
```

Comment:
Sorry, I didn't mean to turn off lint checks for these files, but just to ignore any errors that you have not changed (lint errors doesn't block the merge on PR for us) .